includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1916,6 +1921,17 @@ public function wc_setup_recommended() {
 						'plugins'     => array( array( 'name' => __( 'MailChimp for WooCommerce', 'woocommerce' ), 'slug' => 'mailchimp-for-woocommerce' ) ),
 					) );
 				endif;
+
+				if ( $this->should_show_facebook() ) :
+					$this->display_recommended_item( array(
```

Comment:
I think coding standards sniffs will want the array on it's own line.

Reply:
Hmm, not seeing this in the list of errors in Travis CI: https://travis-ci.org/woocommerce/woocommerce/jobs/470207185